math/fuzz_test.go

```diff
@@ -4,6 +4,33 @@ import (
 	"testing"
 )
 
+func FuzzNewIntFromStringSize(f *testing.F) {
```

Comment:
This should be improved in a follow-up. What is pressing now is to release a v1.1.2 of math that does not break.